frontend/src/components/chats/components/add-member-form/add-member-form.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { useDispatch } from 'react-redux';
+import { useForm } from 'react-hook-form';
+// import { yupResolver } from '@hookform/resolvers/yup';
+import clsx from 'clsx';
+import {
+  InputColor,
+  ButtonType,
+  ButtonColor,
+  ButtonStyleType,
+} from 'common/enums';
+// import { message as messageValidationSchema } from 'validation-schemas';
+import { IMember } from 'common/interfaces';
+import { Button } from 'components/common';
+import { SelectAsync } from '../../components';
+import { ChatsActionCreator } from 'store/slices';
+
+import styles from './styles.module.scss';
+
+enum MemberPayloadKey {
+  MEMBER = 'member',
+}
+
+interface IMemberPayload {
+  [MemberPayloadKey.MEMBER]: IMember | null;
+}
+
+interface Props {
+  className?: string;
+}
+
+const AddMemberForm: React.FC<Props> = ({ className }) => {
+
+  const DEFAULT_VALUES: IMemberPayload = {
+    [MemberPayloadKey.MEMBER]: null,
+  };
```

Comment:
Move default values to `frontend/src/components/chats/components/add-member-form/common/constants/`